climateconnect/climateconnect
# fix: hide header on scroll down and show on scroll up

## Description
Resolve #727

Hide header(navbar) when the user scrolls down on mobile and shows when the user scrolls up. Larger screens remain the same as they were.

## Test plan

- Go to [climate connect](https://climateconnect.earth/)
- Reduce your window size if you're not on mobile.
- Scroll down(navbar should be hidden at this point)
- scroll up(navbar shows up)

**On Scroll down**

![Screenshot (239)](https://user-images.githubusercontent.com/70102539/160473004-40157b99-b18f-427d-b15e-778910744dba.png)

**On Scroll up**

![Screenshot (240)](https://user-images.githubusercontent.com/70102539/160473039-8468b0b3-693b-4d64-b043-d7e0ed0c681b.png)



## Before landing

1. PR has meaningful title
2. `yarn lint` passes (frontend)
3. `yarn format` passes (frontend) -- No, I got the below error Instead tho I follow code pattern to format

> yarn run v1.22.17
warning ..\..\..\package.json: No license field
$ prettier --write '**/*.{js,jsx,json,md}'
[error] No files matching the pattern were found: "'**/*.{js,jsx,json,md}'".
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

5. `make format` passes (backend)

## Extra Steps Taken

- Remove fixed height given to header. I removed it cause it's giving extra height to the header element.
- Reduce padding of header element on mobile.
- Reduce logo size on mobile.
